test/logic/stake_boss_logic_test.rb

```diff
@@ -122,7 +162,7 @@ def create_validators
   end
 
   test 'guard_stake_account_boss_does_not_have_stake_authority' do
-    address = '2tgq1PZGanqgmmLcs3PDx8tpr7ny1hFxaZc2LP867JuS'
+    address = '2tgq1PZGanqgmmLcs3PDx8tpr7ny1hFxaZc2LP867JuSa'
```

Comment:
I know now this is not the proper way of fixing it as Solana has varied address length. I've already fixed this for ticket https://www.pivotaltracker.com/story/show/177580592